smoke-tests/apps/TelemetryProcessors/src/smokeTest/resources/applicationinsights-mask-log-body.json

```
@@ -0,0 +1,55 @@
+{
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100
+  },
+  "preview": {
+    "processors": [
+      {
+        "type": "log",
+        "body": {
+          "toAttributes": {
+            "rules": [
+              "userId (?<redactedUserId>[\\da-zA-Z]+)"
```

just a style preference in this particular case

```suggestion
              "userId (?<redactedUserId>[0-9a-zA-Z]+)"
```